### PR TITLE
chore: update amm, lending and bridge

### PR DESCRIPTION
**Pull request by danielsimao**

### CODE REVIEW

File: docs/guides/bridge.md [hunk 1]
```diff
@@ -192,7 +185,7 @@ You can check the status of your issue request in the Transactions view in the *
 
 If your Bitcoin transaction has enough confirmations but has not been executed by a Vault yet, click on the issue request that is "Pending". This will open a modal, where you will see a **"Claim KBTC"** (or **"Claim IBTC"**) button. To finalize the Issue process and claim your IBTC/KBTC, either wait for a Vault to auto-execute your request, or click **"Claim"** yourself.
 
-![Tansactions page](../_assets/img/guide/interlay-bridge-issue-claim.png)
+![Tansactions page](../_assets/img/guide/bridge-issue-complete-tx.png)
```
Comment (danielsimao) [inline docs/guides/bridge.md:188]: Im not sure about this image @nud3l ´. Previously there was a modal with claim ibtc but im not able to reach that. Only this

File: docs/guides/custom-fees.md [hunk 2]
```diff
@@ -0,0 +1 @@
+# Custom Fees Guide
```
Comment (danielsimao) [inline docs/guides/custom-fees.md:1]: @nud3l what exactly do we want to add here?

Comment (nud3l) [reply docs/guides/custom-fees.md:1]: Just a small screenshot about the fee selector component and how to use it. I'd imagine this page to be very short with a single screenshot and the steps to select the currency.

File: docs/guides/bridge.md (hunk 1 above)
Comment (nud3l) [reply docs/guides/bridge.md:188]: You can reach it when you send less BTC than you requested. Would be great if you could update that!

File: docs/guides/custom-fees.md (hunk 2 above)
Comment (danielsimao) [reply docs/guides/custom-fees.md:1]: Done

File: docs/guides/bridge.md (hunk 1 above)
Comment (danielsimao) [reply docs/guides/bridge.md:188]: Done

File: docs/guides/custom-fees.md [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+
+Our dapp no longer limits the user on using solely the parachain governance token (INTR/KINT) to pay transaction fees. There is now available a set of assets to cover for transaction fees and possibly many more in the near future.
```
Comment (nud3l) [inline docs/guides/custom-fees.md]:
```suggestion
Our dapp no longer limits the user to using solely the Interlay (and Kintsugi) native token (INTR/KINT) to pay transaction fees. Any asset listed in the AMM pools can now be used to pay for transaction fees.

Under the hood, the selected fee token (e.g., DOT) is swapped into the native token (e.g., INTR) to cover fees.
```

File: docs/guides/custom-fees.md [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+
+You can now find a select option on every transaction form across the dapp.
```
Comment (nud3l) [inline docs/guides/custom-fees.md]:
```suggestion
You can select the fee token on every transaction via a "Fee token" selector on each part of the app where you can make a transaction.
```

File: docs/guides/custom-fees.md [hunk 5]
```diff
@@ -0,0 +1,19 @@
...
+
+![Custom Fees Select](../_assets/img/guide/custom-fees-select.png)
+
+When you click the select, a modal with the available options for pays fees is shown.
```
Comment (nud3l) [inline docs/guides/custom-fees.md]:
```suggestion
When you click inside the "Fee token" selector, a modal with the available options for pays fees is shown. This modal shows your balance of each token (not the fees you would pay in the token).
```

File: docs/guides/custom-fees.md [hunk 6]
```diff
@@ -0,0 +1,19 @@
...
+
+![Custom Fees Modal](../_assets/img/guide/custom-fees-select-modal.png)
+
+When you select the desired token to cover for fees, an estimation is ran only when you fulfil the form.
```
Comment (nud3l) [inline docs/guides/custom-fees.md]:
```suggestion
After inserting all information for the transaction and selecting the fee token, the app will give an estimate on how much fees you will have to pay in the selected fee token.
```

File: docs/guides/custom-fees.md [hunk 7]
```diff
@@ -0,0 +1,19 @@
...
+
+![Custom Fees Estimate](../_assets/img/guide/custom-fees-estimate.png)
+
+After the form is fulfilled, a fee estimaste will run. For you to execute the transaction, the necessary funds to cover tge transaction fee needs to be available. If not, make sure you make more funds available or choose another token from the available set.
```
Comment (nud3l) [inline docs/guides/custom-fees.md]:
```suggestion
When completing the inputs, an estimation of the fees is provided. Please ensure that you have enough tokens to cover for fees in at least one of the available fee tokens. If you need to add more tokens, please check the [asset onramp guide](/guides/assets).
```

**Review by nud3l — CHANGES_REQUESTED** [review]
Thanks for adding this!

I've rewrote the fee part a bit.

Can you also add a link to the sidebar please: https://github.com/interlay/interlay.github.io/blob/master/docs/_sidebar.md

The fee guide should go below the governance guide. Thanks!

**Comment by danielsimao** [comment]
@nud3l thank you for the docs coaching :D. Everything is fixed. May i merge?

**Comment by nud3l** [comment]
> @nud3l thank you for the docs coaching :D. Everything is fixed. May i merge?

Can you also address this:

> Can you also add a link to the sidebar please: https://github.com/interlay/interlay.github.io/blob/master/docs/_sidebar.md
> The fee guide should go below the governance guide. Thanks!

**Review by nud3l — APPROVED** [review]

**Pull request merged**